store/kv/kv.go

```diff
@@ -102,6 +103,22 @@ func measureLow(ctx context.Context, name string, f func() error) {
 		metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets)
 		defer metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets).Start().Stop()
 	}
+	if config.DefaultConfig.DatadogTrace.Enabled {
+		spanOpts := []tracer.StartSpanOption{
```

Comment:
Should we move this in a helper method? `GetSpanOptions() []tracer.StartSpanOption`

Reply:
+1! (store/kv/tracing.go?) 

Also, I'd at least define const for these presets in said file. Configurability probably not required.

Reply:
I did that, not in kv package, but in metrics (same place as the tagging logic).